front/app/components/admin/ActionForm/Fields/CustomField/index.tsx

```diff
@@ -60,19 +66,29 @@ const CustomField = ({ field, phaseId, action }: Props) => {
         </Box>
         {!disableEditAndDelete && (
           <Box display="flex">
-            <Button
-              icon="edit"
-              buttonStyle="text"
-              p="0"
-              m="0"
-              mr="28px"
-              onClick={(e) => {
-                e.preventDefault();
-                setIsModalOpen(true);
-              }}
+            <Tooltip
+              content={formatMessage(
+                messages.contactGovSuccessToAccessAddingAQuestion
+              )}
+              disabled={isPermissionsCustomFieldsAllowed}
             >
-              {formatMessage(messages.edit)}
-            </Button>
+              <Box>
+                <Button
+                  icon="edit"
+                  buttonStyle="text"
+                  p="0"
+                  m="0"
+                  mr="28px"
+                  disabled={!isPermissionsCustomFieldsAllowed}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    setIsModalOpen(true);
+                  }}
+                >
+                  {formatMessage(messages.edit)}
```

Comment:
Hmm, I think deleting also needs to be also restricted as this will still allow them to delete default fields, therefore allowing them to create a set of custom permission fields